src/js/plugin/sdk/services/MarathonService.js

```diff
@@ -0,0 +1,23 @@
+
+import config from "../../../config/config";
+import ajaxWrapper from "../../../helpers/ajaxWrapper";
+import Utils from "../utils";
+
+export default class MarathonService {
+
+  static request(opts = {}) {
```

Comment:
Definitely not a blocker but  you could use destructing to extract `resource`, `method` as well as the `data` as illustrated in the following snippet:

```js
 static request({resource, method="GET", data}) {
    // ... 
    return ajaxWrapper({
         url: `${config.apiURL}${resource}`,
         method: method,
         data: opts.data
       });
  }
```

Reply:
This is awesome! Will do it. This will make the code much simpler!